dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -380,15 +378,15 @@ Path findClassLocation(String className) {
   }
 
   /**
-   * Returns a set of {@link JavaClass}es which have entries in the {@code jar} through {@link
+   * Returns a set of {@link JavaClass}es which have entries in the {@code entry} through {@link
    * #classRepository}.
    */
-  private ImmutableSet<JavaClass> listClasses(Path jar) throws IOException {
+  private ImmutableSet<JavaClass> listClasses(ClassPathEntry entry) throws IOException {
     ImmutableSet.Builder<JavaClass> javaClasses = ImmutableSet.builder();
 
     ImmutableList.Builder<String> corruptedClassFileNames = ImmutableList.builder();
 
-    for (String classFileName : listClassFileNames(jar)) {
+    for (String classFileName : listClassFileNames(entry)) {
```

Comment:
`entry.getClassFileNames()`? That is, should we move the listing into the `ClassPathEntry` class?